### ISSUE TITLE
Parsing Failed.

### ISSUE THREAD

**Issue by jdefelice** [jdarling/ingredientparser#1]
When trying to parsing ingredients a noticed it failed to parse "2 teaspoons toasted sesame oil, divided".

I would submit a test but I'm not familiar with the process. I've added this to the test file.

```
  '2 teaspoons toasted sesame oil, divided': {
    amount: '2',
    unit: 'Teaspoon',
    name: 'toasted sesame oil, divided'
  } 
```

{ amount: '2', name: 'aspoons toasted sesame oil, divided' }
      1) Should parse 2 teaspoons toasted sesame oil, divided

**Comment by jdarling** [comment]
I'll take a look, glad someone besides me is finding this lib useful :)

**Comment by jdarling** [comment]
I've pushed a fix for this but unfortunately my npm is jacked right now.  Soon as I get npm fixed I'll publish, but you should be able to do a git clone and get the updates.

Thanks for the test case, that made it much easier to find the problem.

Problem was in the reToWords it wasn't taking into account that you could have a word that began with to (facepalm).  Minor tweak to the regular expression and a change to getAmount resolved it easily once I knew where I was looking.

Will close the bug once I can publish to NPM.

**Comment by jdarling** [comment]
And published as 0.0.3 fully backwards compatible.
